build.gradle

```diff
@@ -50,6 +48,7 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        maven { url "https://oss.sonatype.org/content/repositories/snapshots/"}
```

Comment:
We definitely can't depend on snapshots, that can break any future build…

Reply:
Alright, so should we just ignore ```ObsoleteLintCustomCheck``` until new versions of ButterKnife and Timber is released?

Reply:
I'm not a fan of JitPack, but that could be an option here, or just wait couple days for Jake to properly release new versions :)

Reply:
Maybe if you create an issue and ask nicely we'll get those published faster.

Reply:
Looks like ButterKnife 9.x is far from release https://github.com/JakeWharton/butterknife/milestone/5